Author: Tollef Fog Heen <tfheen@debian.org>
Description: update is called from the Debian maintainer scripts and
 prints non-appropriate output.  Change that.
Forwarded: not-needed

Index: b/bin/update
===================================================================
--- a/bin/update	2010-05-24 17:50:50.000000000 +0200
+++ b/bin/update	2010-05-24 17:51:02.000000000 +0200
@@ -334,37 +334,6 @@
     # BAW: Is this still necessary?!
     mlist.Save()
     #
-    # check to see if pre-b4 list-specific templates are around
-    # and move them to the new place if there's not already
-    # a new one there
-    #
-    # There is a problem with the following if the list name is equal
-    # to a language code. All the templates get moved from templates/xx
-    # to lists/xx. To avoid this, test the list name, and if it is 'en'
-    # or matches a name in the messages directory, just skip this step.
-    if (mlist._internal_name <> 'en' and
-        mlist._internal_name not in os.listdir(mm_cfg.MESSAGES_DIR)):
-        # It's okay to move the templates.
-        tmpl_dir = os.path.join(mm_cfg.PREFIX, "templates")
-        list_dir = mm_cfg.LIST_DATA_DIR
-        b4_tmpl_dir = os.path.join(tmpl_dir, mlist._internal_name)
-        new_tmpl_dir = os.path.join(list_dir, mlist._internal_name)
-        if os.path.exists(b4_tmpl_dir):
-            print _("""\
-- This list looks like it might have <= b4 list templates around""")
-            for f in os.listdir(b4_tmpl_dir):
-                o_tmpl = os.path.join(b4_tmpl_dir, f)
-                n_tmpl = os.path.join(new_tmpl_dir, f)
-                if os.path.exists(o_tmpl):
-                    if not os.path.exists(n_tmpl):
-                        os.rename(o_tmpl, n_tmpl)
-                        print _('- moved %(o_tmpl)s to %(n_tmpl)s')
-                    else:
-                        print _("""\
-- both %(o_tmpl)s and %(n_tmpl)s exist, leaving untouched""")
-                else:
-                    print _("""\
-- %(o_tmpl)s doesn't exist, leaving untouched""")
     #
     # Move all the templates to the en language subdirectory as required for
     # Mailman 2.1
@@ -669,12 +638,12 @@
                 'Mailman/HyperDatabase.py', 'Mailman/pipermail.py',
                 'Mailman/smtplib.py', 'Mailman/Cookie.py',
                 'bin/update_to_10b6', 'scripts/mailcmd',
-                'scripts/mailowner', 'mail/wrapper', 'Mailman/pythonlib',
+                'scripts/mailowner', 'Mailman/pythonlib',
                 'cgi-bin/archives', 'Mailman/MailCommandHandler'):
         remove_old_sources(mod)
     listnames = Utils.list_names()
     if not listnames:
-        print _('no lists == nothing to do, exiting')
+#        print _('no lists == nothing to do, exiting')
         return
     #
     # for people with web archiving, make sure the directories
